test/features/DebugSession.test.ts

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import * as assert from "assert";
+import { DebugConfiguration, DebugSession, debug } from "vscode";
+import { ensureEditorServicesIsConnected } from "../utils";
+
+// Create a fixture that mocks the DebugSessionFeature
+// and provides a way to call the onDidTerminateDebugSession event
+
+describe("Basic Debug Test", () => {
+    before(async () => {
+        // Change the vscode setting debug.toolBarLocation to hidden
+        // This is to prevent the debug toolbar from appearing in the test
+        // and causing the test to fail due to the dialog manager not wanting to show dialogs
+        // await workspace.getConfiguration("debug").update("toolBarLocation", "hidden");
+        await ensureEditorServicesIsConnected();
+    });
+
+    //FIXME: Test currently failing due to https://github.com/microsoft/vscode-test/issues/207
```

Comment:
```suggestion
    // TODO: Test currently failing due to https://github.com/microsoft/vscode-test/issues/207
```

So it shows up in my highlighter 😆

Reply:
I suggest `Gruntfuggly.todo-tree` :). No worries tho.